src/map/script.cpp

```diff
@@ -11647,40 +11647,38 @@ BUILDIN_FUNC(getareadropitem)
  *------------------------------------------*/
 BUILDIN_FUNC(enablenpc)
 {
-	const char *str = script_getstr(st,2);
-	if (npc_enable(str,1))
-		return SCRIPT_CMD_SUCCESS;
-
-	return SCRIPT_CMD_FAILURE;
-}
+	npc_data* nd = nullptr;
+	int flag = 0;
```

Comment:
Guess the assignment can stay here since your checks below doesn't have a final "else". Is this the default value you want to hold (IE: disablenpc) if nothing is matched?

Reply:
>Is this the default value you want to hold (IE: disablenpc) if nothing is matched?

Yes it's the idea